### ISSUE TITLE
prototext parsing error with "original_layer"

### ISSUE THREAD

**Issue by JaeseungYeom** [LLNL/lbann#727]
Prototext parsing error
```
[libprotobuf ERROR /p/lscratchh/yeom2/LBANN-fork/lbann-external-reader/build/gnu.Release.quartz.llnl.gov/protobuf/src/src/google/protobuf/text_format.cc:307] Error parsing text-format lbann_data.LbannPB: 235:21: Message type "lbann_data.TargetReconstruction" has no field named "original_layer"
```
It is from
https://github.com/LLNL/lbann/blob/ae9b30d8e88b1c45c44794d0dc0245d2ddb7978f/model_zoo/models/molecular_autoencoder_candle_pilot2/model_conv_molecular_autoencoder_pilot2.prototext#L235

"original_layer" is not a member of TargetReconstruction.
In addition, input layer does not set "target_mode". Is this deprecated?
Which model prototext should I use?
I am trying to use this because it was used by the last external reader testing with modified convolution dimensions.

**Comment by timmoon10** [comment]
The Pilot2 model is updated in PR #676, although I'm not sure if it works. I'm not sure where the dataset is located.

**Comment by JaeseungYeom** [comment]
I suspect that the data is under /p/lscratchh/brainusr/datasets/cancer/pilot2, but I am not able to access due to permission. However, I copied some files from ray and put them under /p/lscratchh/brainusr/datasets/cancer/pilot2-pub although I am not sure if they are the same or not.

**Comment by JaeseungYeom** [comment]
I found a way to get around the deprecated prototext issue. So, I can keep going for the time being.

**Comment by timmoon10** [comment]
Closed with #676.
